### PR TITLE
Topbar with information

### PR DESCRIPTION
**Pull request by harshvitra**
Content is to be replaced with real content

### CODE REVIEW

File: src/features/common/Layout/Topbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    const [showTopBar, setshowTopBar] = React.useState(true);
+    const { t, ready } = useTranslation(['common']);
+
+    return (
+        <div className={'topbarContainer'} style={{display:showTopBar ? 'flex' : 'none'}}>
+            <p>
+                {t('common:tobBarText')}
+                <a href="https://waldrekord.plant-for-the-planet.org/" target="_blank">
+                {t('common:clickHere')}
+                </a>
```
Comment (norbertschuler) [inline src/features/common/Layout/Topbar/index.tsx:20]: You should just make the whole text clickable :-)

**Comment by sagararyal** [comment]
Todo:

@sarveshpro 

TopBar notices can be added as json in www5 project.
We could think of integrating this into Webapp at some later time.

www5.planet.../topbar/tenantkey/messages.json

```
{
  "de":{
  "text": "Endspurt beim SAT.1 Waldrekord. Pflanz mit - wir schaffen die Million!",
  "ctaText": "Click Here",
  "ctaLink": "https://example.com"
  },
  "en":{
  "text": "Endspurt beim SAT.1 Waldrekord. Pflanz mit - wir schaffen die Million!",
  "ctaText": "Click Here",
  "ctaLink": "https://example.com"
  }
}
```

**Comment by sarveshpro** [comment]
Not working because of CORS issue

File: next.config.js [hunk 2]
```diff
@@ -118,6 +118,7 @@ module.exports = withPlugins([[withBundleAnalyzer], [withSourceMaps]], {
     VERCEL_URL: process.env.VERCEL_URL,
     SITE_IMAGERY_API_URL: SITE_IMAGERY_API_URL,
     WIDGET_URL: process.env.WIDGET_URL,
+    TOPBAR_API_URL: process.env.TOPBAR_API_URL,
```
Comment (norbertschuler) [inline next.config.js:121]: Please add values to https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/.env.production and https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/.env.development and https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/.env.local.example so we have a default value and do not need to set the env variable in all of our deployments.

Comment (sarveshpro) [reply next.config.js:121]: Yes, but this url won't work, will have to find some other solution

Comment (sarveshpro) [reply next.config.js:121]: https://30c3f8e4.planet-ads.pages.dev/topbar/ten_I9TW3ncG/messages.json
gives CORS error when fetched in the webapp

Comment (norbertschuler) [reply next.config.js:121]: @sarveshpro isn't there a possibility to add a header, e.g. `access-control-allow-origin: *`, to allow fetching it?

**Pull request closed without merging**